ompp-run-ubuntu/Dockerfile

```diff
@@ -0,0 +1,76 @@
+# Docker image to run openM++ models on latest Ubuntu LTS
+#
+# https://github.com/openmpp/docker/blob/master/ompp-run-ubuntu/
+#
+# Examples of build and arguments default values:
+#   sudo docker build -t openmpp/openmpp-run:ubuntu .
+#   sudo docker build -t openmpp/openmpp-run:ubuntu --build-arg OMPP_USER=ompp .
+#   sudo docker build -t openmpp/openmpp-run:ubuntu --build-arg OMPP_GROUP=ompp .
+#   sudo docker build -t openmpp/openmpp-run:ubuntu --build-arg OMPP_UID=1999 .
+#   sudo docker build -t openmpp/openmpp-run:ubuntu --build-arg OMPP_GID=1999 .
+#
+# Examples of run:
+#   sudo docker run \
+#     -v $HOME/models:/home/models \
+#     -e OMPP_USER=models -e OMPP_GROUP=models -e OMPP_UID=$UID -e OMPP_GID=`id -g` \
+#     openmpp/openmpp-run:ubuntu \
+#     ./MyModel
+#
+#   sudo docker run \
+#     -v $HOME/models:/home/models \
+#     -e OMPP_USER=models -e OMPP_GROUP=models -e OMPP_UID=$UID -e OMPP_GID=`id -g` \
+#     openmpp/openmpp-run:ubuntu \
+#     mpiexec -n 2 MyModel_mpi -OpenM.SubValues 16
+#
+#   sudo docker run -it openmpp/openmpp-run:ubuntu bash
+#
+#   sudo docker run \
+#     -v $HOME:/home/${USER} \
+#     -e OMPP_USER=${USER} -e OMPP_GROUP=`id -gn` -e OMPP_UID=$UID -e OMPP_GID=`id -g` \
+#     -it openmpp/openmpp-run:ubuntu \
+#     bash
+
+FROM ubuntu:22.04
+
+# disable debconf terminal input
+ARG DEBIAN_FRONTEND=noninteractive
+
+# update base image
+RUN apt-get update
```

Comment:
All these `apt` related RUN commands can be consolidated to 1 layer. https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments 
Will save on layers.